curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-pyramid-generator/660f1bf673487ae0bb25b900.md

```diff
@@ -0,0 +1,46 @@
+---
+id: 660f1bf673487ae0bb25b900
+title: Step 32
+challengeType: 0
+dashedName: step-32
+---
+
+# --description--
+
+Your loop should now run eight times. Inside the body of the loop, print the value of the `i` iterator and see what happens.
+
+# --hints--
+
+You should use `console.log()`.
+
+```js
+assert.match(code, /console\.log/)
+```
+
+You should log the value of `i`.
+
+```js
+assert.match(code, /console\.log\(\s*i\s*\)/);
+```
+
+You should log the value of `i` in your `for` loop. Don't forget your semi-colon.
+
+```js
+assert.match(code, /for\s*\(\s*let\s+i\s*=\s*0;\s*i\s*<\s*count;\s*i\s*=\s*i\s*\+\s*1\s*\)\s*\{\s*console\.log\(\s*i\s*\);\s*\}/);
+```
```

Comment:
semi colons should be optional

````suggestion
You should log the value of `i` in your `for` loop.

```js
assert.match(code, /for\s*\(\s*let\s+i\s*=\s*0;\s*i\s*<\s*count;\s*i\s*=\s*i\s*\+\s*1\s*\)\s*\{\s*console\.log\(\s*i\s*\);?\s*\}/);
```
````